### ISSUE TITLE
Persona 3 Reload

### ISSUE THREAD

**Issue by Burdpal** [tmplshdw/cloud_config_workaround#32]
Hello!

Much like Persona 5 Royal, Steam Deck settings and Desktop settings are shared due to bad Steam cloud functionality. May we have a script to fix this? Thank you!

**Comment by Burdpal** [comment]
The settings directory is located here for Windows:

C:\Users\USERNAME\AppData\Local\P3R\Saved\Config\WindowsNoEditor\GameUserSettings

When I come back from work I can investigate where the Steam Deck directory is located.

**Comment by gdiazbanuelos** [comment]
You can view your Steam cloud saves for Persona 3 Reload at https://store.steampowered.com/account/remotestorageapp/?appid=2161700
Can you send the list of files that are being synced for Persona 3 Reload?

For my Persona 5 Royal steam cloud saves I see:
-  SEGA/P5R/Steam/[STEAMID]/savedata/DATA01/DATA.DAT 
-  SEGA/P5R/Steam/[STEAMID]/savedata/SYSTEM/SYSTEM.DAT 

Hopefully Persona 3 Reload uses a similar method to P5R. Knowing specifically what cloud save files are being used can help get this issue fixed.

**Comment by Burdpal** [comment]
SEGA/P3R/Steam/76561197978341286/SaveData001.sav

**Comment by gdiazbanuelos** [comment]
Is that the only file? If so, then it means the save data and the system data like graphics options are tied together. I am not sure if much can be done. Usually games sync the two as separate files and this script overrides the settings file.

**Comment by Burdpal** [comment]
Regrettably this was the only file.

I was afraid of this... when playing this game on my break, I noticed that loading a save did not respect the game settings I set in the main menu. You're 100% correct: the save file itself houses the graphics settings, which complicates matters as that is the file being shared on the cloud.

Shamefully I'll just reconfigure graphics settings every time I switch devices. I still appreciate your feedback and help, and I still like using this script for P5R and Sekiro. Keep up the good work.
